### PR TITLE
Add social media management

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Review by starbelly — APPROVED** [review]

**Review by pablocostass — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Just waiting on a decision for joint merge+tweet, to get this going...

File: _config.yml [hunk 1]
```diff
@@ -66,114 +66,53 @@ languages:
...
-#    sponsor_url: http://www.erlef.org/
-#    url:
-#    image: sponsors_2020/eef-logo.svg
-#
-#  - name: Free Tickets for Code Mesh V
-#    sponsor: Code Sync
-#    sponsor_url: https://codesync.global/
-#    url:
-#    image: prizes_2020/code_mesh.jpg
+  - name: $2400 Amazon Gift Card
```
Comment (starbelly) [inline _config.yml:112]:
```suggestion
  - name: $2400 Amazon Gift Card(s)
```

Comment (paulo-ferraz-oliveira) [reply _config.yml:112]: I'm Okay with whatever, here. We have decided only one person will distribute the prizes any way :) (and it's not us muahahaha)

**Comment by paulo-ferraz-oliveira** [comment]
Conflicts. Resolving...

**Comment by paulo-ferraz-oliveira** [comment]
Squashing and merging...

**Pull request merged**